### PR TITLE
REL: prep for SciPy 1.9.1

### PR DESCRIPTION
**Pull request by tylerjereddy**
* we're probably going to need SciPy `1.9.1` :)

### CODE REVIEW

File: doc/source/release.1.9.1.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+.. include:: ../release/1.9.1-notes.rst
```
Comment (tupui) [inline doc/source/release.1.9.1.rst:1]: Can I propose to make the change to not have a need for this extra file?

**Review by rgommers — APPROVED** [review]
What optimism:) Thanks Tyler!

File: doc/source/release.1.9.1.rst (hunk 1 above)
Comment (rgommers) [reply doc/source/release.1.9.1.rst:1]: at least not in this branch. in `main` yes you can propose it. I personally get some use out of all the release notes being in a separate directory, but maybe that can be done without needing these one-line files.

Comment (tupui) [reply doc/source/release.1.9.1.rst:1]: I would just like to clean `doc/source`. NumPy has a single place for release files in `doc/source/release` for instance.

**Pull request merged**